public/locales/el/modules/dashdot.json

```diff
@@ -4,6 +4,9 @@
     "description": "",
     "settings": {
       "title": "Ρυθμίσεις για το widget Dash",
+      "dashName": {
+        "label": "Dash. Ονομα"
```

Comment:
Do not commit translations, only english. Crowdin will translate the other languages

Reply:
I understand, I'll revert all the changes except for the en locale

Reply:
@manuel-rw should I remove the entire object from the json, or just the value of label